packages/pn-commons/src/components/CopyToClipboard.tsx

```diff
@@ -1,7 +1,7 @@
-import { useState, useEffect } from 'react';
-import { Button, Link, SxProps, Theme, Tooltip } from '@mui/material';
-import ContentCopyIcon from '@mui/icons-material/ContentCopy';
 import CheckIcon from '@mui/icons-material/Check';
```

Comment:
I'm not sure about this component. We should use the mui-italia component and not this. Do the check